### PR TITLE
[B] Various fixes

### PR DESCRIPTION
**Pull request by 1aurend**
Resolves:
- WDP-735
- WDP-740
- WDP-745

### CODE REVIEW

**Comment by 1aurend** [comment]
@jen-castiron I only implemented placeholders for the metadata fields specifically mentioned in WDP-735. Let me know if you'd like me to add them consistently across all fields.

File: packages/frontend/components/composed/article/ArticleMetadataBlock/ArticleMetadataBlock.tsx [hunk 1]
```diff
@@ -8,16 +8,58 @@ import { ArticleMetadataBlockFragment$key } from "@/relay/ArticleMetadataBlockFr
 import MetadataFactory from "components/factories/MetadataFactory";
 import { MetadataProperty } from "components/layout";
 import { NamedLink, PrecisionDate } from "components/atomic";
+import ContributorName from "components/composed/contributor/ContributorName";
 
 export default function ArticleMetadataBlock({ data }: Props) {
   const entity = useMaybeFragment(fragment, data);
+  const authors = entity?.contributions?.edges?.filter(
+    ({ node }) => node.role?.toLowerCase() === "author"
+  );
```
Comment (1aurend) [inline packages/frontend/components/composed/article/ArticleMetadataBlock/ArticleMetadataBlock.tsx:17]: This doesn't seem optimal, but I wasn't sure how else to get just authors.

Comment (jen-castiron) [reply packages/frontend/components/composed/article/ArticleMetadataBlock/ArticleMetadataBlock.tsx:17]: This is fine for now - we may want some API change in the future, to get contributions by role

**Review by jen-castiron — APPROVED** [review]
Thank you for jumping on these bugs! 🎷 🐛

**Pull request merged**